### PR TITLE
Test:  clean up after test execution

### LINKED ISSUE #8337: Tests:  tests not cleaning up
Some package signing-related tests open handles on files but do not close them properly. Thus, these tests fail to clean up after themselves.

### PR DESCRIPTION
**Pull request by dtivel**
Fix https://github.com/NuGet/Home/issues/8337.

Dispose disposables.

### CODE REVIEW

File: build/config.props [hunk 1]
```diff
@@ -66,8 +66,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <NUGET_PFX_PATH>$(MSBuildThisFileDirectory)\..\keys\NuGetKey.snk</NUGET_PFX_PATH>
-    <MS_PFX_PATH>$(MSBuildThisFileDirectory)\..\keys\35MSSharedLib1024.snk</MS_PFX_PATH>
```
Comment (dtivel) [inline build/config.props:70]: [`MsBuildThisFileDirectory`](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-reserved-and-well-known-properties?view=vs-2019) already includes the trailing backslash.

Comment (nkolev92) [reply build/config.props:70]: Were the paths being resolved correctly prior to this change?

**Review by nkolev92 — APPROVED** [review]
LGTM.

File: build/config.props (hunk 1 above)
Comment (dtivel) [reply build/config.props:70]: Yes, just an extra slash.

    /keyfile:E:\A\_work\68\s\build\\..\keys\35MSSharedLib1024.snk

**Pull request merged**